components/widgets/menu/menu.scss

```diff
@@ -14,7 +14,7 @@
 
         ul {
             margin: 0;
-            padding: 0;
+            padding: 8px 0;
```

Comment:
This should be handled by component SCSS, since changing the base-component has side-effects for other implementations

Reply:
@asaadmahmood